Add blog announcing nightly playgrounds launch (#3404)
opensearch-project/project-website: _community_members/sayaligaikawad.md

redirect_from: '/authors/sayaligaikawad/'
---

> Sayali Gaikawad is a software development engineer at OpenSearch Project at Amazon. She works with Engineering Effectiveness team at OpenSearch that concentrates on release processes, build and test tools, and re-usable infrastructure for the OpenSearch project making the life of developers a little bit easier.

Reviewer: ```suggestion
Sayali Gaikawad is a software development engineer at AWS working on the OpenSearch Project. She works with the Engineering Effectiveness team, which concentrates on release processes, build and test tools, and reusable infrastructure for the OpenSearch Project, making developers' lives a little bit easier.

```
